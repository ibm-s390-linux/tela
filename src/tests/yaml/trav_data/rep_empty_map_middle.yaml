map1:
replace:
map2:
