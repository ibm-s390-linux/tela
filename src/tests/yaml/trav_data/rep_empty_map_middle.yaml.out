Before (root=non-null):
==================================
  map1:
  replace:
  map2:
==================================

Callback:
==================================
map1                                    : map1:
replace                                 : replace:
*** Replacing node
map2                                    : map2:
==================================

After (root=non-null):
==================================
  map1:
  replacement
  map2:
==================================
